vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -709,12 +715,6 @@ void LedgerImpl::ContributionCompleted(
   bat_contribution_->ContributionCompleted(
       result,
       contribution->Clone());
-
-  // TODO(https://github.com/brave/brave-browser/issues/7717)
-  // rename to ContributionCompleted
-  ledger_client_->OnReconcileComplete(
```

Comment:
Is this removal intentional?

Reply:
yes, it was moved into `contribution.cc` - `Contribution::ContributionCompleted`